library/globals.inc.php

```diff
@@ -1258,6 +1258,12 @@ function gblTimeZones() {
             '0',                              // default
             xl('Option to save method of document storage.')
         ),
+        'document_drag_drop' => array(
+            xl('Drag n Drop Document Uploading'),
+            'bool',
+            '0',
```

Comment:
Agree to default to 1 (or if works really well, then would drop this global altogether). Let me (or @robertdown get back to you on this after testing.